stacker/session_cache.py

```diff
@@ -1,16 +1,25 @@
 import json
 import os
-import logging
 import botocore
+import boto3
 
 
 def get_session(region):
+    """Creates a boto3 session with a cache
+
+    Args:
+        region (str): The region for the session
+
+    Returns:
+        Object: A  boto3 session with credential
```

Comment:
This actually returns a `boto3.session.Session` object, right?  See https://github.com/remind101/stacker/blob/52b4508ab2f030f5127774afdd50d2f27206ef7b/stacker/stack.py#L63 for an example of how you can document those.